models/ed/R/met2model.ED2.R

```diff
@@ -149,9 +149,7 @@ met2model.ED2 <- function(in.path, in.prefix, outfolder, start_date, end_date, l
     ## determine GMT adjustment lst <- site$LST_shift[which(site$acro == froot)]
 
     ## extract variables
-    lat  <- eval(parse(text = lat))
-    lon  <- eval(parse(text = lon))
-    sec  <- nc$dim$time$vals
```

Comment:
I renamed this to `tdays` (line 152 in the new version) to make the units clearer.